barretenberg/cpp/src/barretenberg/vm/tests/avm_kernel.test.cpp

```diff
@@ -614,6 +616,8 @@ void negative_test_incorrect_ia_kernel_lookup(OpcodesFunc apply_opcodes,
     auto& ta = trace.at(1);
 
     ta.main_ia = incorrect_ia;
+    // memory trace should only have one row for these tests as well, so first row has looked-up val
```

Comment:
Fair enough we are not supposed to modify unit tests every day and so this is fine for this use case.